README.md

```diff
@@ -15,7 +15,7 @@ API documentation is available at [http://hexdocs.pm/phoenix](http://hexdocs.pm/
 
 ## Contributing
 
-We appreciate any contribution to Phoenix, so check out our [CONTRIBUTING.md](CONTRIBUTING.md) guide for more information. We usually keep a list of features and bugs [in the issue tracker][1].
+We appreciate any contribution to Phoenix, that's following our [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md), so check out our [CONTRIBUTING.md](CONTRIBUTING.md) guide for more information. We usually keep a list of features and bugs [in the issue tracker][1].
```

Comment:
I just changed it again due to more feedback so please make you grab the latest. :)